rocAL/rocAL/include/meta_data/meta_data.h

```diff
@@ -242,7 +242,7 @@ class MetaDataBatch
         *this += *other;
         return this;
     }
-    virtual std::shared_ptr<MetaDataBatch> clone() = 0;
+    virtual std::shared_ptr<MetaDataBatch> clone(bool copy_contents = true) = 0;
```

Comment:
I see from master_graph.cpp that we are creating an empty instance if there's no contents in this. But is there an use case for the else condition. If not, can you make the if condition the default?

Reply:
There is use for else condition